app/commands/v2/import.rb

```diff
@@ -0,0 +1,134 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+
+      def self.call(payload)
+        logger.debug "#{self} called with payload:\n#{payload}"
```

Comment:
It was for consistency with the BaseCommand, but now we are just using its implementation.